lib/sinks/file-sink.js

```diff
@@ -1,7 +1,35 @@
-const fs = require("fs-promises");
+'use strict';
+
+const fs = require('fs').promises;
+const StringBuilder = require('../string-builder');
+
+
+// add chaining syntax returning (this) in StringBuilder
+// here we maybe can iterate asynchronously
+const buildMessage = (loggingLevel, messageParts, entities) => {
+  const spaceCount = 2;
+  const sb = new StringBuilder();
+  for (let i = 0; i < messageParts.length - 1; i++) {
+    sb.append(messageParts[i]);
+    if (i === messageParts.length - 1) {
+      sb.append(`\n#${loggingLevel}`);
```

Comment:
why using # symbol? It means nothing in the terms of filesystem and is only useful for telegram sink.
You can change format to "LOGGING_LEVEL [DateTime]: Logging message"